cfme/tests/infrastructure/test_project_quota.py

```diff
@@ -0,0 +1,118 @@
+import fauxfactory
+import pytest
+
+from cfme import test_requirements
+from cfme.infrastructure.provider.rhevm import RHEVMProvider
+from cfme.infrastructure.provider.virtualcenter import VMwareProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.provisioning import do_vm_provisioning
+from cfme.utils.generators import random_vm_name
+
+from widgetastic.utils import partial_match
```

Comment:
Required: this should be moved to the top:
import fauxfactory
import pytest
from widgetastic.utils import partial_match